pkg/api/vlabs/validate.go

```diff
@@ -663,25 +660,13 @@ func (a *Properties) Validate(isUpdate bool) error {
 			case Swarm:
 			case SwarmMode:
 			case Kubernetes:
-				var version string
-				if a.HasWindows() {
-					version = common.RationalizeReleaseAndVersion(
-						a.OrchestratorProfile.OrchestratorType,
-						a.OrchestratorProfile.OrchestratorRelease,
-						a.OrchestratorProfile.OrchestratorVersion,
-						true)
-				} else {
-					version = common.RationalizeReleaseAndVersion(
-						a.OrchestratorProfile.OrchestratorType,
-						a.OrchestratorProfile.OrchestratorRelease,
-						a.OrchestratorProfile.OrchestratorVersion,
-						false)
-				}
-				if version == "" {
```

Comment:
Let's put this empty string check back, and remove the `common.AllKubernetesWindowsSupportedVersions[version]` check